src/lib.rs

```diff
@@ -809,6 +848,26 @@ pub enum OnlineWalletSubCommand {
         )]
         tx: Option<String>,
     },
+    /// Produce a proof of reserves
+    #[cfg(feature = "reserves")]
+    ProduceProof {
+        /// Sets the message
+        #[structopt(name = "MESSAGE", long = "message")]
+        msg: String,
+    },
+    /// Verify a proof of reserves for our wallet
+    #[cfg(feature = "reserves")]
+    VerifyProof {
+        /// Sets the PSBT to verify
+        #[structopt(name = "BASE64_PSBT", long = "psbt")]
+        psbt: String,
+        /// Sets the message to verify
+        #[structopt(name = "MESSAGE", long = "message")]
+        msg: String,
+        /// Sets the number of block confirmations for UTXOs to be considered. If nothing is specified, 6 is used.
+        #[structopt(name = "CONFIRMATIONS", long = "confirmations")]
```

Comment:
I think a better way to specify the default for the cli is like this, it will also show in the help so you don't need to mention it in the above docs line:
```suggestion
        #[structopt(name = "CONFIRMATIONS", long = "confirmations", default_value = "6")]
```